config/locales/zh-TW.yml

```diff
@@ -59,6 +29,7 @@ zh-TW:
       help:
         redirect_uri: '每個網址必須只有一行'
         native_redirect_uri: '使用 %{native_redirect_uri} 作本地端測試'
+        scopes: 'Separate scopes with spaces. Leave blank to use the default scopes.'
```

Comment:
thank you review.
you mean https://github.com/shingo-nakanishi/doorkeeper-i18n/commit/cdffebf042916f0f95aee773a0e5161f38cde09f (I made new commit)?